common-topics/direct-vs-psp.md

```diff
@@ -48,27 +53,38 @@ and
 [eCom](https://developer.vippsmobilepay.com/docs/APIs/ecom-api)
 APIs have some functionality that is not available in the PSP API:
 
-1. Higher success rate.
+1. Direct integration allows you to have more control over the payment process
```

Comment:
What do we actually mean about the payment process? You could argue that with a PSP solution you have more control of the actual payment process if refering to the authorization process since you can choose your own acquirer etc. If we mean user flow, since customization is also mentioned, it is true to some extend. But it is also possible to get custome setups through a PSP (you probably just need to pay for it). 

I suggest that we change 'payment process' to 'integration'. Because that is what they get conrtrol of

Reply:
Wasn't this wording suggested by you..? 😅

Reply:
Updated! 🎉